ArduCopter/Parameters.h

```diff
@@ -370,8 +370,12 @@ class Parameters {
 
         // 254,255: reserved
 
+        // Semi auto mode param
+        k_param_auto_man_alt = 256, // activate semi auto mode add a2sAndres
+        k_param_enbl_crs_lock, // activate enabled course locked func add a2sAndres
```

Comment:
Can't use this space, you need to use G2. Also remove the `add a2sAndres`.

Reply:
revised and changed, use g2 added in the line 601,line. To add the parameters I use, use the wiki tutorial http://ardupilot.org/dev/docs/code-overview-adding-a-new-parameter.html, for which I suggest that this section be updated to be clarified when you should use the k_pararms and the G2 param, as well as add the g2 params